src/components/Resource.jsx

```diff
@@ -74,12 +74,16 @@ export const Resource = (props) => {
       <StyledLeftASide>
         <LeftTBox>
           <LeftBox>
-            <LeftLogo><IoIosBookmark /></LeftLogo>
+            <LeftLogo>
+              <IoIosBookmark />
+            </LeftLogo>
             <LeftName>My resources</LeftName>
           </LeftBox>
 
           <LeftBox onClick='PesourcePopup()'>
-            <LeftLogo><HiBellAlert /></LeftLogo>
+            <LeftLogo>
+              <HiBellAlert />
```

Comment:
LOL. My brain goes against the programmer norms with these changes. I see the code's  structure clearer the previous way. 

My instructor explained this to me because her brain sees your way clearer too. 

I always forget this when working with other people. Majority Rules. I'll try to remember to stop changing these for everyone else.

Reply:
No problem, it keeps changing because my vscode is configured to change styles onSave...